source/frontend/src/features/leases/detail/LeasePages/tenant/AddLeaseTenantForm.test.tsx

```diff
@@ -91,7 +91,7 @@ describe('AddLeaseTenantForm component', () => {
       component: { getByText },
     } = await setup({});
 
-    const tenantButton = getByText('Select Tenant(s)');
```

Comment:
This doesn't seem correct, since the user may select multiple tenants on this screen.

Reply:
This is just the test matcher for the text. The text itself has not been changed